### PR TITLE
new ToolCard component, RTL styling for ToolCard and ResourceCard

### PR DESCRIPTION
**Pull request by Erin007**
Previously, the RTL layout for jumbo `ResourceCard`s was wonky. 
screen shot 2017-08-16 at 5 44 20 pm

The wonkiness was caused because RTL styles needed to be different based on whether the `ResourceCard` had an image or not and whether it was jumbo or not. Thus it seemed this component was trying to handle too much.  I separated this component into `ResourceCard` and `ImageResourceCard`. 

`ResourceCard` in RTL: 
rtl tools

`ResourceCard` should be used if the use case does NOT have an image and you want 3 cards to a row (at full desktop width). Alternatively, `ImageResourceCard` can be used if you want a larger component that fits two to a row and includes a picture. Separating these components makes their styling more straightforward and their use more specific. 

`ImageResourceCard` in RTL: 
screen shot 2017-08-16 at 6 21 16 pm

### CODE REVIEW

**Comment by Erin007** [comment]
@tanyaparker

File: apps/src/templates/studioHomepages/ResourceCard.jsx [hunk 1]
```diff
@@ -15,99 +16,52 @@ const styles = {
...
+    image: PropTypes.string.isRequired,
     isRtl: PropTypes.bool.isRequired
   };
 
   render() {
 
-    const { isJumbo, title, description, buttonText, link, image, isRtl } = this.props;
+    const { title, description, buttonText, link, image, isRtl } = this.props;
+    const localeStyle = isRtl ? styles.rtl : styles.ltr;
+    const uncoverImage = isRtl ? {marginRight:160} : {marginRight:0};
```
Comment (breville) [inline apps/src/templates/studioHomepages/ResourceCard.jsx]: I think we should put all CSS in the styles array and then merge in additional styles if necessary, just to keep CSS out of JS.

**Comment by breville** [comment]
We use these image-less blocks for more than just tools (see https://github.com/code-dot-org/code-dot-org/pull/16868 for screenshots) so I wonder if we can name them something more related to their appearance.  Maybe SomethingBlock, but I'm not sure what that Something is.

**Comment by tanyaparker** [comment]
What about `ResourceCard` and `ImageResourceCard`?

**Pull request merged**